sdk/communication/AzureCommunicationCalling/CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Release History
 
+## 2.0.0-beta.1 (2021-07-14)
+### New features
+- `Recording` and `Transcription` features are decoupled from `Call` object and now can only be used via extensions.
+  For e.g. 
+```
+            let recordingFeature = self.call!.api(RecordingFeature.self)
+            recordingFeature.delegate = self.callObserver
+```
+
+### Breaking changes
+- `didChangeRecordingState` and `didChangeTranscriptionState` are moved out of `CallDelegate` into `RecordingFeatureDelegate` and into `TranscriptionFeatureDelegate`.
+
+**More documentation on extensions and the breaking change can be found [here](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/voice-video-calling/calling-client-samples?pivots=platform-ios).**
+
+*NOTE: Version previously released `1.2.0-beta.1` and this version only has the difference in version number and this version will be available in cocoapods.*
+
+### Bug fixes
+- [iOS] Not triggering calldidChangeState if using createCallAgentWithCallKitOption. And not able to accept the call in App killed state https://github.com/Azure/Communication/issues/316.
```

Comment:
This format should follow the example in the updated guidance:
```
- Issue where AccountName on BlobUriBuilder would not be populated
  for non-IP style Uris ([#8638](https://github.com/Azure/azure-sdk-for-net/issues/8638))
```
We should remove the [iOS] tag and make sure the URL is represented as shown above.

Reply:
[iOS] tag comes from the bug title. But i can change the link format

Reply:
The changelog should not just replicate the bug title, as that is not always a logical explanation of the bug. This should be a concise description of the issue.